examples/dem/3d-bouncing-particle/bouncing-particle.prm

```diff
@@ -0,0 +1,95 @@
+# Listing of Parameters
+# ---------------------
+# --------------------------------------------------
```

Comment:
What is the differnece between this bouncing particles and the numbered ones?

Reply:
" bouncing-particle.prm " will be remove.